### ISSUE TITLE
The extension goes to a different page when I am scrolling an element with a horizontal overflow

### ISSUE THREAD

**Issue by artisticfox8** [selleronom/PrecisionTouchPadSwipe#12]
**Describe the bug**
When I scroll code comments on github.com or stackoverflow.com the extension navigates back or forward. When my mouse is inside such code comments, I simply want to scroll them horizontally, not go to a different page.

**To Reproduce**
1. Go to any page which has elements with overflow-x:  auto, for example on github
2. Try to scroll that element to see it with a touchpad
3. Actual behaviour: page navigates

**Expected behavior**
The extension allows horizontal scrolling on overflow-x:  auto elements and doesn't call window.history.back(); 
Instead it only navigates back when the element.scrollLeft == 0  and only navigates forward when element.scrollLeft == element.scrollLeftMax

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Firefox
 - Version 99 Developer edition

**Comment by artisticfox8** [comment]
I already have fixed it, will publish a PR soon

**Comment by selleronom** [comment]
Yes, had some issues updating the add-on due to "Unsafe assignment to innerHTML on options/options.js line 5 and line 7".
Are they needed?

**Comment by artisticfox8** [comment]
> Yes, had some issues updating the add-on due to "Unsafe assignment to innerHTML on options/options.js line 5 and line 7". Are they needed?

If the addons team has issue with that, you can swap it to innerText just fine
